Tests/PackageRegistryToolTests/PackageRegistryToolTests.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import XCTest
+@testable import PackageRegistryTool
+
+final class PackageRegistryToolTests: XCTestCase {
```

Comment:
These new tests can go into `CommandsTests/PackageRegistryToolTests.swift`.

Reply:
Roger that, done!